polling_stations/apps/data_finder/views.py

```diff
@@ -222,6 +222,14 @@ def get(self, request, *args, **kwargs):
         if "postcode" not in kwargs or kwargs["postcode"] == "":
             return HttpResponseRedirect(reverse("home"))
 
+        # Log to firehose
+        entry = settings.POSTCODE_LOGGER.entry_class(
+            postcode=kwargs["postcode"],
+            dc_product=settings.POSTCODE_LOGGER.dc_product.wdiv,
+            **self.request.session.get("utm_data"),
+        )
+        settings.POSTCODE_LOGGER.log(entry)
+
```

Comment:
I assume the reason this is going here rather than the `LogLookUpMixin` class is so that we only log postcodes. In which case it makes sense. Once we're happy with firehose I guess we can extract this out if we want.